libminifi/include/core/ProcessSession.h

```diff
@@ -39,6 +39,7 @@
 #include "provenance/Provenance.h"
 #include "utils/gsl.h"
 #include "ProcessorMetrics.h"
+#include "utils/ValueIdProvider.h"
```

Comment:
This was left out from https://github.com/apache/nifi-minifi-cpp/pull/1447/commits/43fd546047aa2e9312ccefaf6f2625a5c1639993 and causes compile errors